round4/team-06.md

```diff
@@ -0,0 +1,140 @@
+13. As a researcher conducting literature reviews, I want to understand the main ideas and concepts of the papers I’m reviewing fast, so that I can more productively select the most relevant papers.
+14. As a student preparing for standardized tests, I want to learn important vocabulary words and their definitions from reading passages and study materials, so that I can improve my performance on vocabulary-based questions.
+15. As a professional attending industry conferences and seminars, I want to quickly grasp the meaning of specialized terms and acronyms used in presentations and discussions, so that I can actively participate in conversations and networking opportunities.
+16. As a journalist covering diverse topics, I want to quickly understand the meaning of technical terms and specialized terminology used in interviews and press releases, so that I can accurately report on complex issues and communicate effectively with my audience.
+
+## Notes
+
+There is a lot of functionality that could be added on to this extension, depending on if we want to make a product that is extremely good at one thing, or a product that has many features all related to summarization and generative AI analysis of webpage content. 
+We plan to use OpenAI’s API for summarization, given it’s reliability and extremely strong developer support.
+
+## References & Inspiration
+
+OpenAI’s ChatGPT
+
+## Technical Details
+
+### User Interface
+
+Chrome Elements:
+- Action Button
+  - User clicks the extension action button to open the sidebar
+- Sidebar
+  - User uses the extension by interacting with UI components within the sidebar including:
+  - Scrollable text boxes
+  - Dropdown menus
+  - Hoverable buttons
+- [Potential Expansion] Separate webpage
+  - User can choose to open the sidebar in its own webpage (full tab)
+
+[Figma file](https://www.figma.com/file/ZP70Xo4z1mLCbY2uHYD8fi/LazyReader?type=design&node-id=1%3A2&mode=design&t=uT3QaqEEoUQaPr83-1
+)
+
+<i>(Quick overview screenshots attached)</i>
+
+### API, Libraries, and Frameworks
+
+- [OpenAI] (https://platform.openai.com/docs) : We will use the OpenAI API to query the content we want to summarize.
+- [ReactJS] (https://react.dev) : Utilize ReactJS for the frontend to create components.
+- [Some JS library for web scraping] (TBD): We plan to either find a JS library to help us with web scraping or simply draw from targeted DOM elements. 
+
+### Data Storage
+
+- Utilize local storage to store the user’s API Key and the previous summary/keywords/questions.
+
+## Project Management
+
+### Collaboration and Task Allocation
+
+Leader: Emma Lee
+Manager: Rohan Kamat
+Remaining Team Members: Annie Wang, Kevin Lee
+
+- [Kevin] Repository Setup and establishing Chrome extension foundation
+  - Kevin will spearhead the initial setup of our project repository, which includes configuring the foundation, setting up the basic structure and backbone. 
```

Comment:
What does foundation, structure and backbone really mean? I think you need to be precise when you talk about a technical project that you are doing ( this goes even with tech interviews for example )  maybe replace those words with some action item like taking care of merging conflict, integrating dependencies, or creating the script and the IU component ...Or whatever Kevin will be exactly doing.  Those are better terms to show what Kevin will do in this project.